Sources/XMTPiOS/Client.swift

```diff
@@ -172,13 +215,40 @@ public final class Client: Sendable {
 			rustClient: client
 		)
 
-		return try Client(address: address, privateKeyBundleV1: v1Bundle, apiClient: apiClient)
+		let v3Client: FfiXmtpClient?
+
+		if options.enableAlphaMLS == true && options.api.env == .local {
+			let dbURL = URL.documentsDirectory.appendingPathComponent("xmtp-\(address).db3")
+			v3Client = try await LibXMTP.createClient(
+				logger: XMTPLogger(),
+				host: GRPCApiClient.envToUrl(env: apiClient.environment),
+				isSecure: apiClient.environment != .local,
+				db: dbURL.path,
+				encryptionKey: nil,
+				accountAddress: address,
+				legacyIdentitySource: .static,
+				legacySignedPrivateKeyProto: try v1Bundle.toV2().identityKey.serializedData()
+			)
+
+			try await v3Client?.registerIdentity(recoverableWalletSignature: nil)
+		} else {
+			v3Client = nil
+		}
+
```

Comment:
This scenario is actually the one where text to sign is probably going to be most important but I don't see the signer work here?

Reply:
Maybe I misunderstood the text to sign parts. This is the flow where we just load keys we already have so we don't have access to a signer?

Reply:
Ya thats the breaking change we discussed last week. If they don't have a signer and also dont have V3 keys they can't create a v3 client. The first time requires a signer. So you'll need to have them pass a signer in which would be a forced log out.

Reply:
The only way to tell if they have v3 keys or not is to check if they need to sign or not.